src/src/Array.hpp

```diff
@@ -23,71 +23,91 @@
 #ifndef ARRAY_HPP_
 #define ARRAY_HPP_
 
+// Source includes
+#include "helpers.hpp"
+#include "ArrayView.hpp"
+#include "bufferManipulation.hpp"
+
+// TPL includes
+#include <RAJA/RAJA.hpp>
 
+// System includes
 #include <iostream>
 #include <limits>
 
-#include "ArrayView.hpp"
-#include "helpers.hpp"
-#include "ChaiVector.hpp"
-#include "SFINAE_Macros.hpp"
-
-
 
 namespace LvArray
 {
 
-
 /**
  * @class Array
  * @brief This class provides a multi-dimensional array interface to a vector type object.
- * @tparam T    type of data that is contained by the array
+ * @tparam T type of data that is contained by the array
  * @tparam NDIM number of dimensions in array (e.g. NDIM=1->vector, NDIM=2->Matrix, etc. )
+ * @tparam PERMUTATION a camp::idx_seq containing the values in [0, NDIM) which describes how the
+ *         data is to be laid out in memory. See RAJA::PERM_IJK etc.
```

Comment:
I think we should have an explanation for this independent of RAJA::PERM_IJK. The RAJA explanations aren't very intuitive.

Reply:
Done.